### ISSUE TITLE
Add `-First` switch to the `Switch` statement

### ISSUE THREAD

**Issue by iRon7** [PowerShell/PowerShell#17016]
### Summary of the new feature / enhancement

Selecting just the first occurrence in a [`switch` statement](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_switch) is a quiet common usage (which also appears from the given examples).
It would be nice to have a `-First` (or something similar, as e.g. `-AutoBreak`) switch to automatically break after the first match. 
This will prevent repeating myself with `; break` statements or possibly forget it.

In other words, current statement:

```PowerShell
switch (2)
{
    1 {"It is one."; Break}
    2 {"It is two." ; Break }
    3 {"It is three." ; Break }
    4 {"It is four." ; Break }
    Default {"No match found."}
}
```

Wishful thinking:

```PowerShell
switch (2) -First
{
    1 {"It is one." }
    2 {"It is two." }
    3 {"It is three." }
    4 {"It is four." }
    Default {"No match found."}
}
```

**Comment by iRon7** [comment]
@ritikBhandari,

> Can i work on this?

I didn't answer your question as I do not exactly know how everything is prioritized and dispatched, but I wonder if you ever got an answer on this.

**Comment by iRon7** [comment]
This issue <kbd>Needs-Triage</kbd> and either marked with something like <kbd>Resolution-Declined</kbd> or put it on a kind of <kbd>wishlist</kbd> "[*Issue we would like to prioritize, but we can't commit we will get to it yet*](https://github.com/dotnet/runtime/labels?q=wishlist)"
